src/runtime_src/core/pcie/driver/linux/xocl/subdev/icap.c

```diff
@@ -2529,8 +2529,11 @@ static int __icap_download_bitstream_axlf(struct platform_device *pdev,
 
 	/* create the reset of subdevs for both mgmt and user pf */
 	if (num_dev > 0) {
-		for (i = 0; i < num_dev; i++)
-			xocl_subdev_create(xdev, &subdevs[i].info);
+		for (i = 0; i < num_dev; i++) {
+			err = xocl_subdev_create(xdev, &subdevs[i].info);
+			if (err && err != -EEXIST)
+				goto done;
+		}
```

Comment:
I'd suggest to just ignore the error by adding (void) (see line 2526, which is also creating subdevs).